miner/algo_common.go

```diff
@@ -264,14 +264,30 @@ func (envDiff *environmentDiff) commitBundle(bundle *types.SimulatedBundle, chDa
 	bundleSimEffGP := new(big.Int).Set(bundle.MevGasPrice)
 
 	// allow >-1% divergence
-	bundleActualEffGP.Mul(bundleActualEffGP, big.NewInt(100))
+	bundleActualEffGP.Mul(bundleActualEffGP, common.Big100)
```

Comment:
Should this check be done if `enforceProfit` is set?

Reply:
this is EGP enforcement, not necessarily profit enforcement